libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.html

```diff
@@ -0,0 +1,10 @@
+<div #layout class="fd-fixed-card-group--card-layout" cdkDropListGroup>
+    <div class="fd-fixed-card-group--card-layout-column" [cdkDropListData]="column" cdkDropList
+        *ngFor="let column of columns">
+        <ng-container *ngFor="let card of column; let i=index">
+            <div class="fd-fixed-card-group--card" cdkDrag (cdkDragDropped)="dragDrop($event)" [attr.tabindex]="i">
+                <ng-container *ngTemplateOutlet="card?.template"> </ng-container>
+            </div>
+        </ng-container>
+    </div>
+</div>
```

Comment:
Angular always had a performace issue using nested `ngFor`.
How many cards are we going to support here? 
I'm not sure if it's feassible and reasonble particulary in this case but maybe we could flatten it to use only one `ngFor`.

Reply:
There is no limit on number of cards but if we see our usecases, we don't see number of cards to be used more than 10-15 in a page.

"text from spec":
<img width="540" alt="Screenshot 2020-09-22 at 4 06 20 PM" src="https://user-images.githubusercontent.com/57661487/93872304-95450a80-fced-11ea-99b9-f58d9b97e03e.png">

Reply:
Good, in case of 15 cards it won't be an issue I think